src/test/java/pl/project13/maven/git/FileSystemMavenSandbox.java

```diff
@@ -162,21 +144,14 @@ public MavenProject getChildProject() {
     return childProject;
   }
 
-  public File getSandboxDir() { return gitRepoTargetDir.getAbsoluteFile(); }
-
   @NotNull
   private MavenProject createProject(File basedir, String packaging) {
     MavenProject project = new MavenProject();
-    project.setFile(new File(basedir + FILE_SEPARATOR + "pom.xml"));
+    project.setFile(new File(basedir + File.separator + "pom.xml"));
```

Comment:
Now that I see this I think File.pathSeparator was meant for all of these actually